docker/Dockerfile.flux

```diff
@@ -1,5 +1,20 @@
 FROM alpine:3.6
 
+WORKDIR /home/flux
+
+RUN apk add --no-cache openssh ca-certificates tini 'git>=2.3.0'
+
+# Add git hosts to known hosts file so when git ssh's using the deploy
+# key we don't get an unknown host warning.
+RUN mkdir ~/.ssh && touch ~/.ssh/known_hosts && \
+    ssh-keyscan github.com gitlab.com bitbucket.org >> ~/.ssh/known_hosts && \
```

Comment:
We are not using /.ssh/known_hosts anymore, please use the latest master dockerfiles.